src/cli/fleet.toit

```diff
@@ -560,23 +623,42 @@ class FleetWithDevices extends Fleet:
     a device hasn't set its state yet.
   */
   roll-out --diff-bases/List:
-    detailed-devices := {:}
     fleet-devices := devices_
+    device-ids := fleet-devices.map: it.id
 
-    existing-devices := broker.get-devices --device-ids=(fleet-devices.map: it.id)
+    detailed-devices := broker.get-devices --device-ids=device-ids
     fleet-devices.do: | fleet-device/DeviceFleet |
-      if not existing-devices.contains fleet-device.id:
+      if not detailed-devices.contains fleet-device.id:
         ui_.abort "Device $fleet-device.id is unknown to the broker."
 
     pods-per-group := {:}  // From group-name to Pod.
     pods := fleet-devices.map: | fleet-device/DeviceFleet |
       group-name := fleet-device.group
       pods-per-group.get group-name --init=: download (pod-reference-for-group group-name)
 
-    broker.roll-out --devices=existing-devices.values --pods=pods --diff-bases=diff-bases
+    broker.roll-out --devices=detailed-devices.values --pods=pods --diff-bases=diff-bases
 
     ui_.info "Successfully updated $(fleet-devices.size) device$(fleet-devices.size == 1 ? "" : "s")."
 
+    // We need to notify the migrated-from brokers.
+    fleet-file_.migrated-from.do: | server-name |
+      ui_.info "Rolling out on migrated-from broker $server-name."
+      server-config := fleet-file_.servers.get server-name
+      old-broker := Broker
+          --server-config=server-config
+          --cache=cache_
+          --config=config_
+          --ui=ui_
+          --fleet-id=id
+          --organization-id=organization-id
+          --tmp-directory=artemis.tmp-directory
+      // We could filter out devices that were already known in the new broker, but
+      // it's easier and more robust to update all devices.
+      // This also makes it possible to move forward and backward between two brokers.
+      detailed-devices = old-broker.get-devices --device-ids=device-ids
+      old-broker.roll-out --devices=detailed-devices.values --pods=pods --diff-bases=diff-bases
+      ui_.info "Successfully rolled out on migrated-from broker $server-name."
```

Comment:
Successfully rolled out to $old broker (migration in progress).

Reply:
done.